hack/run-e2e-local.sh

```
@@ -1,290 +1,168 @@
-#!/bin/bash
-set -e
+#!/usr/bin/env bash
+set -euo pipefail
 
-# Configuration
-CLUSTER_NAME="automotive-dev-e2e"
-REGISTRY_NAME="kind-registry"
-REGISTRY_PORT="5001"
+NAMESPACE="automotive-dev-operator-system"
```

Shellcheck (0.11.0)

[warning] NAMESPACE appears unused. Verify use (or export if used externally).

(SC2034)